src/utils/apiHelpers.ts

```diff
@@ -24,6 +24,10 @@ const apiBaseUrl = (() => {
 })();
 
 function apiResourceUrl(path: string): string {
+  if (path.startsWith('/myndla-api/v1')) {
+    return 'http://localhost:1349' + path;
+  }
```

Comment:
Dette skal bort, bare jeg som gikk for raskeste og enkleste måten å gå mot både test og lokalt samtidig 😄

Reply:
Så lenge du husker det!!

Reply:
Husket!!